Editor/Core/FoundationConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace StansAssets.Foundation.Editor
+{
+    public static class FoundationConstants
```

Comment:
All public endpoints have to be documented :)

Reply:
summary added